src/gei/Commands/GenerateReclaimCsvCommand.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OctoshiftCLI.GithubEnterpriseImporter.Commands
+{
+    public class GenerateReclaimCsvCommand : Command
+    {
+        internal Func<string, string, Task> WriteToFile = async (path, contents) => await File.WriteAllTextAsync(path, contents);
+
+        private readonly OctoLogger _log;
+        private readonly ITargetGithubApiFactory _targetGithubApiFactory;
+        private readonly EnvironmentVariableProvider _environmentVariableProvider;
+
+        public GenerateReclaimCsvCommand(OctoLogger log, ITargetGithubApiFactory targetGithubApiFactory, EnvironmentVariableProvider environmentVariableProvider) : base("generate-reclaim-csv")
+        {
+            _log = log;
+            _targetGithubApiFactory = targetGithubApiFactory;
+            _environmentVariableProvider = environmentVariableProvider;
```

Comment:
It seems like you're not using the `environmentVariableProvider` so it can be removed from the `ctor`

Reply:
deleted